test/e2e/serial/tests/resource_accounting.go

```diff
@@ -128,7 +129,6 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload resourc
 
 			candidateNodeNames := e2enrt.AccumulateNames(nrtCandidates)
 			// nodes we have now are all equal for our purposes. Pick one at random
-			// TODO: make sure we can control this randomness using ginkgo seed or any other way
```

Comment:
why this comment was removed?

Reply:
because we have the ginkgo seed here:
https://github.com/openshift-kni/numaresources-operator/blob/main/test/e2e/serial/e2e_serial_test.go#L63

Reply:
we do, and we cannot control the seed :) unless we do crazy hacks with the machine clock, which we totally don't want to do.

Reply:
besides, seeding ginkgo repeatedly (which we don't do yet) won't solve the other randomness usage, like iterating over a map like we do in `PopAny`

Reply:
ok thanks for elaborating, I'll add such comments back, it'll be interesting to achieve this randomness